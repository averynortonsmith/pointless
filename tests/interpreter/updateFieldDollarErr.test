
output = [a] |> println

a = {bar = "asdf"} with {
  $.foo = 4
  $.foo = $[0]
}