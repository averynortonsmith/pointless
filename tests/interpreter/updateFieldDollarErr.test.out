-------------------------------------------------------------------------------
Parser Error:

Expected 'Number or String or Name or Label or Bool or LArray or LParen or LBracket', got 'Dollar'
-------------------------------------------------------------------------------

At (line 6 column 11) in 'tests/interpreter/updateFieldDollarErr.test'
  $.foo = $[0]
          ^
